src/test/java/teammates/sqlui/webapi/RegenerateInstructorKeyActionTest.java

```diff
@@ -0,0 +1,166 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.util.Const;
+import teammates.common.util.Const.InstructorPermissions;
+import teammates.common.util.JsonUtils;
+import teammates.storage.sqlentity.Account;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.storage.sqlentity.Instructor;
+import teammates.ui.output.InstructorData;
+import teammates.ui.request.Intent;
+import teammates.ui.webapi.RegenerateInstructorKeyAction;
+import teammates.common.util.EmailType;
+import teammates.common.util.EmailWrapper;
+import teammates.ui.output.RegenerateKeyData;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link RegenerateInstructorKeyAction}.
+ */
+public class RegenerateInstructorKeyActionTest extends BaseActionTest<RegenerateInstructorKeyAction> {
```

Comment:
Hi, no need to do unit tests now, can just migrate existing integration tests to the it package.